qualtricssurvey/private/edit.html

```diff
@@ -24,6 +23,110 @@
               of the page.
           </span>
       </li>
+      
+      <li class="field comp-setting-entry is-set">
+          <div class="wrapper-comp-setting">
+              <label
+                  class="label setting-label"
+                  for="xblock_qualtricssurvey_survey_id"
+              >
+                  Survey ID
+            </label>
+            <input
+                class="input setting-input"
+                id="xblock_qualtricssurvey_survey_id"
+                value="{self.survey_id}"
+                type="text"
+                tabindex="2"
+            >
+          </div>
+          <span
+              class="tip setting-help"
+          >
+              This is the ID that Qualtrics uses for the survey,
+               usually an 8 digit number.
+          </span>
+      </li>
+      
+      <li class="field comp-setting-entry is-set">
+          <div class="wrapper-comp-setting">
+              <label
+                  class="label setting-label"
+                  for="xblock_qualtricssurvey_your_university"
+              >
+                  Your university
+            </label>
+            <input
+                class="input setting-input"
+                id="xblock_qualtricssurvey_your_university"
+                value="{self.your_university}"
+                type="text"
+                tabindex="3"
+            >
+          </div>
+          <span
+              class="tip setting-help"
+          >
+              This is the name of your university.
+          </span>
+      </li>
+      
+      <li class="field comp-setting-entry is-set">
+          <div class="wrapper-comp-setting">
+              <label
+                  class="label setting-label"
+                  for="xblock_qualtricssurvey_link_text"
+              >
+                  Link text
+            </label>
+            <input
+                class="input setting-input"
+                id="xblock_qualtricssurvey_link_text"
+                value="{self.link_text}"
+                type="text"
+                tabindex="4"
+            >
+          </div>
+          <span
+              class="tip setting-help"
+          >
+              This is the text that will link to your survey.
+          </span>
+      </li>
+      
+      <li class="field comp-setting-entry is-set">
+          <div class="wrapper-comp-setting">
+              <label
+                  class="label setting-label"
+                  for="xblock_qualtricssurvey_param_name"
+              >
+                  Parameter Name for UserID
+            </label>
+            <input
+                class="input setting-input"
+                id="xblock_qualtricssurvey_param_name"
+                value="{self.param_name}"
+                type="text"
+                tabindex="5"
+            >
+          </div>
+          <span
+              class="tip setting-help"
+          >
+              This is the name for the User ID parameter in the url.
```

Comment:
Actually, I'd suggest taking a look at the `xblock-utils` repo [1].

It's a helper library that can provide a default Studio view. I haven't used it, but it appears to be a more sophisticated, yet simpler, approach.

I would consider that.

[1] https://github.com/edx/xblock-utils